packages/core/phase3/lib/pncUpdateRequestGenerators/remandGenerator/getResultsForRemand.ts

```diff
@@ -0,0 +1,45 @@
+import type { Offence, Result } from "../../../../types/AnnotatedHearingOutcome"
+import type { PncOperation } from "../../../../types/PncOperation"
+import type { Operation } from "../../../../types/PncUpdateDataset"
+
+import areOrganisationUnitsEqual from "../../../../lib/areOrganisationUnitsEqual"
+import isRecordableOffence from "../../../../phase2/lib/isRecordableOffence"
+import isRecordableResult from "../../../../phase2/lib/isRecordableResult"
+import ResultClass from "../../../../types/ResultClass"
+
+const noAdjudicationResultClasses = [
+  ResultClass.ADJOURNMENT,
+  ResultClass.ADJOURNMENT_PRE_JUDGEMENT,
+  ResultClass.ADJOURNMENT_WITH_JUDGEMENT
+]
+
+const adjudicationResultClasses = [ResultClass.ADJOURNMENT_POST_JUDGEMENT, ResultClass.ADJOURNMENT_WITH_JUDGEMENT]
+
+const IsRemandOperationMatchingResult = (result: Result, operation: Operation<PncOperation.REMAND>) => {
```

Comment:
Think we want this to begin with a lowercase "i", `isRemandOperationMatchingResult`?